frontend/src/components/OperationStepForm/OperationStepForm.tsx

```diff
@@ -147,6 +152,28 @@ export const OperationStepForm: FunctionComponent<OperationStepFormProps> = (pro
     }
   };
 
+  const validateAdvanced = (step: OperationStepMap) => {
+    const options = step.get('query_kwargs') as string;
+    const advanced_query: any = options ? JSON.parse(options) : [];
+    console.log(advanced_query);
+
+    if (advanced_query['validation'].length > 0) {
+      setAlerts({
+        error: `These columns donot exist: ${advanced_query['validation']}`,
+      });
+
+      return false;
+    } else if (advanced_query) {
+      return true;
+    } else {
+      setAlerts({
+        error: 'At least one query filter is required, please prefix columns with "x_"',
```

Comment:
This `x_` prefix is bound to bite you in the butt ... there's lots of IATI columns that have it.

Reply:
Switched to `col_`